### PR TITLE
AR-794 filter no collection result

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/components/Filters/components/LocationsFilter.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ const buildNode = (id, count, t) => ({
 
 const LocationsFilter = props => {
   const cIDs           = useSelector(state => selectors.getEventsByType(state.events)[CT_CONGRESS]);
-  const congressEvents = useSelector(state => (cIDs || []).map(x => mdb.getDenormCollection(state.mdb, x)));
```
Comment (gshilin) [inline src/components/Filters/components/LocationsFilter.js:63]: Here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter#return_value written: A new array with the elements that pass the test. If no elements pass the test, an empty array will be returned. So you shouldn't use the last `|| []`

File: src/components/Pages/PlaylistCollection/Container.js [hunk 2]
```diff
@@ -15,20 +15,20 @@ const PlaylistCollectionContainer = ({ cId, t, cuId }) => {
   const wipMap      = useSelector(state => selectors.getWip(state.mdb));
   const errorMap    = useSelector(state => selectors.getErrors(state.mdb));
   const cWindow     = useSelector(state => selectors.getWindow(state.mdb));
-  const collections = useSelector(state => cWindow?.data?.map(id => selectors.getDenormCollection(state.mdb, id)));
```
Comment (gshilin) [inline src/components/Pages/PlaylistCollection/Container.js:18]: same

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/Pages/PlaylistCollection/Container.js (hunk 2 above)
Comment (davgur) [reply src/components/Pages/PlaylistCollection/Container.js:18]: fixed

File: src/components/Filters/components/LocationsFilter.js (hunk 1 above)
Comment (davgur) [reply src/components/Filters/components/LocationsFilter.js:63]: fixed

**Review by gshilin — APPROVED** [review]

**Pull request merged**